WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -109,6 +108,13 @@ private extension ItemListView {
         .padding(.horizontal, Constants.bannerCardPadding)
     }
 
+    private var bannerHintAndLearnMoreText: Text {
+        Text(Localization.headerBannerHint + " ") +
```

Comment:
It's crazy how easy it is just to append two `Text` objects on `SwiftUI`. Much harder to archieve the same effect on `UIKit`.

Reply:
I wasn't looking forward to attributed strings, definitely an improvement :D